tokens/models/tokens/avalanche_c/tokens_avalanche_c_erc20.sql

```diff
@@ -237,5 +237,10 @@ FROM (VALUES
         ,(0xefd6aa06eb95e0ab23de9ac0977d870888b89a71, 'MICRO', 18)
         ,(0x913c61ec3573e5e4ee6488552535fb1be84ff2ac, 'XAV', 18)
         ,(0x609268b9c47c7be0a8d77ae93c31d2bf6859d175, 'LONG', 18)
-        ,(0xa25eaf2906fa1a3a13edac9b9657108af7b703e3, 'ggAVAX', 18)    
+        ,(0xa25eaf2906fa1a3a13edac9b9657108af7b703e3, 'ggAVAX', 18)
+        ,(0x184ff13B3EBCB25Be44e860163A5D8391Dd568c1, 'KIMBO', 18)
+        ,(0x4f94b8aef08c92fefe416af073f1df1e284438ec, 'WOLF', 18)
+        ,(0x13e793e76c38a6362f3f712c4ae365a9174f9b56, 'FOX', 18)
+        ,(0xf7b084572bd638d8282f4933143a8eac94482f28, 'RPG', 18)
+        ,(0xe8385cecb013561b69beb63ff59f4d10734881f3, 'GEC', 18)
```

Comment:
Maybe some of these tokens have already been added?
i.e.
https://github.com/duneanalytics/spellbook/blob/fd988751375efeb0b439d5c299f57828b817fb1e/tokens/models/tokens/avalanche_c/tokens_avalanche_c_erc20.sql#L223

Reply:
good catch. A query of cryptokorys i found (tredy new tokens) wasn't  pulling the symbols of those addys so figured it was because they were missing from tokens.erc20 .

now that i'm thinking about it.. I may have been the person who added a couple of those previously. (=